brave/brave-core
# Use component for NTP Background images on desktop

Resolves https://github.com/brave/brave-browser/issues/10278

Use component for ntp background images on desktop.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch brave and check first NTP is below spensor's photo
2. Check `NTP Background Images` is loaded in brave://components
3. If not loaded, every NTP will display below bundled image.
4. If already loaded, NTP will show different images from components
5. Toggle `Show Sponsored Images` or `Show Background Images` option and check BG images are shown properly based on changed option

<img width="1155" alt="Screen Shot 2021-10-28 at 6 17 19 PM" src="https://user-images.githubusercontent.com/6786187/139226820-da18cfd8-a445-4ee4-9750-8d1f7745e4f5.png">
